coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -367,3 +367,86 @@ def set_segment_messages_metrics(self, segment_id, messages_metrics):
         :type metrics_map: core_data_modules.data_models.metrics.MessagesMetrics
         """
         self.get_segment_messages_metrics_ref(segment_id).set(messages_metrics.to_firebase_map())
+
+    def compute_segment_coding_progress(self, segment_id, messages=None, force_recount=False):
+        """
+        Compute and return the progress metrics for a given dataset.
+
+        This method will initialise the counts in Firestore if they do not already exist.
+
+        :param segment_id: Id of a segment.
+        :type segment_id: str
+        :param messages: list of core_data_modules.data_models.message.Message, defaults to None
+        :type messages: core_data_modules.data_models.message.Message | None
+        :param force_recount: If True, forces the recount of progress metrics. Defaults to False
+        :type force_recount: bool, optional
+        :return: Messages metrics.
+        :rtype: core_data_modules.data_models.metrics.MessagesMetrics
+        """
+        if not force_recount:
```

Comment:
Do we need this argument? From memory I think we always set this to true as we currently use it

Reply:
I've confirmed that and removed the arg and related code.
I've also removed the log statement.